docs: Implement Herodotos. (#1)
wiris/thoth: docs/USAGE.md

> # Using Thoth

Reviewer: En algún lugar de este archivo pondría que dado que Thoth es un un MVP, antes de añadir nuevos productos se debe consultar a los Leads de Ingeniería.
También algún texto y enlace a las limitaciones que da Github a la hora de consumir su "raw content"

Author: Hecho

> Thoth acts as a `plain text repository` hosted by Github. If you application needs to consume a file content with no logic involved, Thoth can help you.

Reviewer 2: ```suggestion
Thoth acts as a `plain text repository` hosted by Github. If your application needs to consume a file content with no logic involved, Thoth can help you.
```

To do so, follow the next steps:

>   1. Create a folder with a significative name in the root directory of the project (if it does not exist yet). No rules regarding naming folder have been set yet, so just use the common sense.

Reviewer 2: ```suggestion
  1. Create a directory with a meaningful name in the root directory of the project (if it does not exist yet). No rules regarding naming directory have been set yet, so just use the common sense.
```

>   2. Create as many `plain text files` as needed in that folder. How you name those files is up to you. Again, apply the common sense.

Reviewer 2: ```suggestion
  2. Create as many `plain text files` as needed in that directory. How you name those files is up to you. Again, apply the common sense.
```

  3. Go to the repository [page in GitHub](https://github.com/wiris/thoth), browse to the file you need to consume from your application, and open it as "raw". This way, GitHub will show you it's content in a new tab.
  4. Copy that tab's URL: that's the one your code should use to consume the file content.

## How to add, modify, or remove files

You can use GitHub itself to perform any of these actions. Anyway, if you feel comfortable with Git, you should do any of these actions just be pulling and pushing the necessary commits.

## About code review

> TODO: Define PR policies.

Reviewer: Si la modificación es sobre archivos ya existentes el PR se limita a los miembros del squad, si el cambio es por añadir un nuevo producto, el PR debe englobar a los leads de ingeniería

Reviewer: La manera de englobar a todos los leads de ingeniería es con @ wiris/engineering-leads (lo he separado de la @ para no generar un aviso a todos con este comentario)

Author: Hecho
